openghg/data/config/objectstore/defaults.json

```diff
@@ -0,0 +1,258 @@
+            },
+            "met_model": {
+                "type": [
+                    "str"
+                ]
+            }
+        }
+    },
+    "flux": {
+        "required": {
+            "species": {
+                "type": [
+                    "str"
+                ]
+            },
+            "source": {
+                "type": [
+                    "str"
+                ]
+            },
+            "domain": {
+                "type": [
+                    "str"
+                ]
+            }
+        },
+        "optional": {
+            "database": {
+                "type": [
+                    "str"
+                ]
+            },
+            "database_version": {
+                "type": [
+                    "str"
+                ]
+            },
+            "model": {
+                "type": [
+                    "str"
+                ]
+            }
+        }
+    },
+    "flux_timeseries": {
+        "required": {
+            "species": {
+                "type": [
+                    "str"
+                ]
+            },
+            "source": {
+                "type": [
+                    "str"
+                ]
+            },
+            "region": {
+                "type": [
+                    "str"
+                ]
+            }
+        },
+        "optional": {
+            "database": {
+                "type": [
+                    "str"
+                ]
+            },
+            "database_version": {
+                "type": [
+                    "str"
+                ]
+            },
+            "model": {
+                "type": [
+                    "str"
+                ]
+            }
+        }
+    },
+    "surface": {
+        "required": {
+            "species": {
+                "type": [
+                    "str"
+                ]
+            },
+            "site": {
+                "type": [
+                    "str"
+                ]
+            },
+            "sampling_period": {
+                "type": [
+                    "str"
+                ]
+            },
+            "station_long_name": {
+                "type": [
+                    "str"
+                ]
+            },
+            "inlet": {
+                "type": [
+                    "str"
+                ]
+            },
+            "instrument": {
+                "type": [
+                    "str"
+                ]
+            },
+            "network": {
+                "type": [
+                    "str"
+                ]
+            },
+            "source_format": {
+                "type": [
+                    "str"
+                ]
+            },
+            "data_source": {
+                "type": [
+                    "str"
+                ]
+            },
+            "icos_data_level": {
+                "type": [
+                    "str"
+                ]
+            },
+            "data_type": {
```

Comment:
Just spotted this after submitting review - I don't think we should have `data_type` here as a required metadata key?

Reply:
The "required keys" for obs surface currently on devel are:
``` 
required_keys = (
                "species",
                "site",
                "sampling_period",
                "station_long_name",
                "inlet",
                "instrument",
                "network",
                "source_format",
                "data_source",
                "icos_data_level",
                "data_type",
            )
```
We probably don't want data type in there, but I guess it's been there for a while.

Reply:
Ok yeah I see it -https://github.com/openghg/openghg/blob/devel/openghg/store/_obssurface.py#L380. Thanks for pointing that out.

For the purposes of keeping this a neutral change then we can leave it for now but would you mind opening an issue for this? Should be a quick fix to remove and I don't believe would have much impact given data types are separated anyway but should do in a separate PR if we're being strict about it.

Reply:
Okay, I added an issue: https://github.com/openghg/openghg/issues/1045